var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -55,6 +57,8 @@ def cmake_args(self):
             ]
             for bundle in bundles:
                 args.append("-DUSE_BUNDLE_{0}:BOOL=OFF".format(bundle))
+            if "+distmesh" in spec:
+                args.append("-DUSE_BUNDLE_Omega_h:BOOL=OFF")
```

Comment:
Just looking through this. Should the bundle be always off with `+distmesh`? Seems to me that it could be added to the list of bundles above…

Reply:
IMO the `bundle` variant is independent of the `distmesh` variant.  So I don't understand  the change above.

Besides `OMEGA_H` should be added to the `bundles` local variable.

Reply:
If I remember correctly, the discussion was (I think @pramodk was in the loop) that we do not want to bundle omega-h with STEPS that's why I removed it. The thinking was that omega-h should rather be treated like PETSc.